PWGHF/HFL/Tasks/taskSingleMuon.cxx

```diff
@@ -171,15 +173,23 @@ struct HfTaskSingleMuonSelection {
     HistogramConfigSpec hTHnPt{HistType::kTHnSparseD, {axispT, axispTGen, axispTDif, axisTrkType}, 4};
     HistogramConfigSpec hTHnEta{HistType::kTHnSparseD, {axisEta, axisEtaGen, axisEtaDif, axisTrkType}, 4};
     HistogramConfigSpec hTHnP{HistType::kTHnSparseD, {axisP, axisPGen, axisPDif, axisTrkType}, 4};
+    HistogramConfigSpec hTHnMuLite{HistType::kTHnSparseD, {axispT, axisEta, axisDCA, axisTrkType, axisChi2MatchMCHMFT}, 5};
 
-    registry.add("hMuBcuts", "", hTHnMu);
-    registry.add("hMuAcuts", "", hTHnMu);
-    registry.add("hPTBcuts", "", hTHnPt);
-    registry.add("hPTAcuts", "", hTHnPt);
-    registry.add("hEtaBcuts", "", hTHnEta);
-    registry.add("hEtaAcuts", "", hTHnEta);
-    registry.add("hPBcuts", "", hTHnP);
-    registry.add("hPAcuts", "", hTHnP);
+    if (histMuonLite) {
+      registry.add("hMuLiteBcuts", "", hTHnMuLite);
+      registry.add("hMuLiteAcuts", "", hTHnMuLite);
+    } else {
+      registry.add("hMuBcuts", "", hTHnMu);
+      registry.add("hMuAcuts", "", hTHnMu);
+      if (histMuonMC) {
+        registry.add("hPTBcuts", "", hTHnPt);
+        registry.add("hPTAcuts", "", hTHnPt);
+        registry.add("hEtaBcuts", "", hTHnEta);
+        registry.add("hEtaAcuts", "", hTHnEta);
+        registry.add("hPBcuts", "", hTHnP);
+        registry.add("hPAcuts", "", hTHnP);
+      }
```

Comment:
I think the `A` for after and `B` for "after" and "before" is not very obvious, especially since A and B suggest that A goes before B. Can you spell the words out in the histogram names to make it clearer?